ui/src/state/storage/upload.tsx

```diff
@@ -20,7 +20,8 @@ export const useFileStore = create<FileStore>((set) => ({
         hostname: endpoint.host,
         path: endpoint.pathname || '/',
       },
-      region: 'global',
+      // this region is necessary for compatibility with other S3 providers (i.e., filebase)
+      region: 'us-east-1',
```

Comment:
we're going to have contention here until we request an actual region in our storage creds and use that.

Reply:
Agreed. Think we should go ahead and just add it?